Sources/OpenId4VP/AuthorizationRequest/AuthorizationRequest.swift

```diff
@@ -0,0 +1,120 @@
+import Foundation
+
+extension Dictionary where Key == String, Value == String {
+    func values(forKeys keys: [String]) -> [String]? {
+        let values = keys.compactMap { self[$0] }
+        return values.count == keys.count ? values : nil
+    }
+}
+
+struct AuthorizationRequest {
+    let clientId: String
+    let presentation_definition: String?
+    let scope: String?
+    let response_type: String
+    let response_mode: String
+    let nonce: String
+    let state: String
+    let response_uri: String
```

Comment:
why all variables are in snakecase

Reply:
Did a rename seems like not updated. Refactored it.